src/config-hardware/mouse-touchpad.md

```diff
@@ -0,0 +1,65 @@
+# Mouse & Touchpad
+
+Configure Mouse and Touchpad settings in `Settings` ➞ `Mouse & Touchpad`.
+
+![Mouse & Touchpad Settings](/images/config-mouse-touchpad/mouse-touchpad-settings.png)
+
+Mouse settings include Mouse Speed, Mouse Acceleration, and Natural Scrolling.
+
+Touchpad settings include enabling or disabling the touchpad entirely, Natural Scrolling, Touchpad Speed, Tap to Click, Two-finger scrolling, Edge Scrolling, and Disable while typing. Users can also set the Right or Left mouse button as the primary button.
```

Comment:
I'm not sure how much value simply listing all of the options in a paragraph is adding, particularly when easier-to-read lists with descriptions are located lower down on the same page.